features/OCM_insights_tab.feature

```diff
@@ -0,0 +1,90 @@
+      And user U1 is part of account (organization) at least one cluster C1
+      And the cluster C1 has not sent any data to the external data pipeline
+     When user U1 selects "Clusters" item from the menu
+     Then new page with list of clusters should be displayed
+      And the page should contain at least one cluster
+      And button "Create cluster" should be made visible
+      And button "Register cluster" should be made visible
+     When user U1 click on cluster name
+     Then a page with detailed information about the selected cluster should be displayed
+      And at least four tab headers should be displayed on that page
+          | Tab name         |
+          | Overview         |
+          | Monitoring       |
+          | Insights Advisor |
+          | Support          |
+     When user U1 clicks on tab named "Insights Advisor"
+     Then new page should be displayed saying "No recommendations to display"
+      And button with text "What is insights?" is displayed on this page
+     When user U1 selects the "Log out" item from the user menu
+     Then the page with title "Log in to your Red Hat account" is displayed
+
+
+  Scenario: Insights tab for cluster with 1 potential security issue identified
+    Given console.redhat.com is accessible
+      And user U1 is already logged in into console.redhat.com
+      And user U1 switch to "OpenShift" page in console.redhat.com (OCM UI)
+      And user U1 is part of account (organization) at least one cluster C1
+      And the cluster C1 sent data to the external data pipeline
+      And 1 potential issue has been identified (ie. one security recommendation has been generated by CCX pipeline)
+     When user U1 selects "Clusters" item from the menu
+     Then new page with list of clusters should be displayed
+      And the page should contain at least one cluster
+      And button "Create cluster" should be made visible
+      And button "Register cluster" should be made visible
+     When user U1 click on cluster name
+     Then a page with detailed information about the selected cluster should be displayed
+      And at least four tab headers should be displayed on that page
+          | Tab name         |
+          | Overview         |
+          | Monitoring       |
+          | Insights Advisor |
+          | Support          |
+     When user U1 clicks on tab named "Insights Advisor"
+     Then new page should be displayed saying "1 potential issue identified"
+      And the page should contains several sections
+      And "Recommendations by category section" should display the following data
+          | Attribute            | Value |
+          | Fault Tolerance      | 0     |
+          | Performance          | 0     |
+          | Security             | 1     |
+          | Service Availability | 0     |
+      And exactly one rule name should be displayed in the third section
+     When user expands the rule by clicking on ">" icon
```

Comment:
or by clicking on name of the recommendations 

Also please s/rule/recommendation. Rule is the condition that we run internally, recommendation is the condition + content.